examples/with-docker/README.md

```diff
@@ -20,6 +20,21 @@ yarn create next-app --example with-docker nextjs-docker
 
 You can view your images created with `docker images`.
 
+### In existing projects
+
+To add support for docker to an existing project, just copy the `Dockerfile` into the root of the project and add the following to the `next.config.js` file:
```

Comment:
```suggestion
To add support for Docker to an existing project, just copy the `Dockerfile` into the root of the project and add the following to the `next.config.js` file:
```

Reply:
Thanks, @balazsorban44 I'm not an English native so this is very helpful.

Reply:
Thanks!